### ISSUE TITLE
Sort Request Log

### ISSUE THREAD

**Issue by jsamuel** [RequestPolicy/requestpolicy#278]
```
imported trac ticket
created: 2011-11-07 00:35:27
reporter: zozo
```

When the request log is open and the page reloaded, I'd like to be able to sort the list based on whether the destination is blocked or not. 

Alternatively, you could have a toggle to display blocked/unblocked destinations.

I will find either option better than having to scroll down what is sometimes a very long list.

**Comment by jsamuel** [comment]
This appears to be a duplicate of #157. Please re-open if it is now. Thanks.
